lms/djangoapps/course_structure_api/v0/views.py

```diff
@@ -290,3 +297,363 @@ class CourseGradingPolicy(CourseViewMixin, ListAPIView):
+          all the descendants.
+          Default is 3.
+
+          Example: navigation_depth=3
+
+    **Response Values**
+
+        * root: The ID of the root node of the course blocks.
+
+        * blocks: A dictionary that maps block usage IDs to a collection of information about each block.
+          Each block contains the following fields.
+
+          * id: (string) The usage ID of the block.
+
+          * type: (string) The type of block. Possible values include course, chapter, sequential, vertical, html,
+            problem, video, and discussion. The type can also be the name of a custom type of block used for the course.
+
+          * display_name: (string) The display name of the block.
+
+          * children: (list) If the block has child blocks, a list of IDs of the child blocks.
+            Returned only if the "children" input parameter is True.
+
+          * block_count: (dict) For each block type specified in the block_count parameter to the endpoint, the
+            aggregate number of blocks of that type for this block and all of its descendants.
+            Returned only if the "block_count" input parameter contains this block's type.
+
+          * block_json: (dict) The JSON data for this block.
+            Returned only if the "block_json" input parameter contains this block's type.
+
+          * block_url: (string) The URL to retrieve the HTML rendering of this block.  The HTML could include
+            CSS and Javascript code.  This URL can be used as a fallback if the custom block_json for this
+            block type is not requested and not supported.
+
+          * web_url: (string) The URL to the website location of this block.  This URL can be used as a further
+            fallback if the block_url and the block_json is not supported.
+
+          * graded (boolean) Whether or not the block or any of its descendants is graded.
+            Returned only if "graded" is included in the "fields" parameter.
+
+          * format: (string) The assignment type of the block.
+            Possible values can be "Homework", "Lab", "Midterm Exam", and "Final Exam".
+            Returned only if "format" is included in the "fields" parameter.
+
+          * responsive_ui: (boolean) Whether or not the block's rendering obtained via block_url is responsive.
+            Returned only if "responsive_ui" is included in the "fields" parameter.
+
+        * navigation: A dictionary that maps block IDs to a collection of navigation information about each block.
+          Each block contains the following fields.
+
+          * descendants: (list) A list of IDs of the children of the block if the block's depth in the
+            course hierarchy is less than the navigation_depth.  Otherwise, a list of IDs of the aggregate descendants
+            of the block.
+
+        * blocks_navigation: A dictionary that combines both the blocks and navigation data.
+
```

Comment:
This is great documentation!

Nit: I assume you'd get back one of blocks, navigation, or blocks_navigation? Perhaps that could be made clear?